org.mixedrealitytoolkit.input/Tests/Runtime/Utilities/BaseRuntimeInputTests.cs

```diff
@@ -63,25 +86,45 @@ protected ControllerLookup CachedLookup
                 return cachedLookup;
             }
         }
+#pragma warning restore CS0618 // Type or member is obsolete
 
+        [Obsolete("Deprecated, please use SetupForXRI3Testing()")]
+#pragma warning disable CS0809 // Obsolete member overrides non-obsolete member
```

Comment:
I did that because I had marked MRTKRigPrefabGuid as obsolete aiming to make easier the later step of removing deprecated elements, but you are right, at this point we are just adding new testing and the existing testing remains valid.  I'll fix this.

Reply:
Removed in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/757/commits/11bc312c268fb172de6932aeaa9836837152d3f2 , resolving.

Reply:
@ms-RistoRK Can you please clarify what is meant by "later step of removing deprecated elements"?  What elements will be eventually be removed?

Reply:
> @ms-RistoRK Can you please clarify what is meant by "later step of removing deprecated elements"? What elements will eventually be removed?

@whebertML :
I refer to the elements that Unity has already been marked as Obsolete and for future removal as part of their new XRI3 package.  In particular, the components associated to XRIController concepts.  For example, in our case that would be the XRController component:

![image](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/84108471/891a9e55-76fc-4544-84ff-e2049a7b7be0)

Such effort is listed as a goal in the [Update MRTK3 to support XRI 3.0] issue (https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/issues/645), in particular:

![image](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/84108471/b91c3eb1-463a-45b5-80c5-bf699abdbff8)

Note:  The removal possibility will also depend on the customers and maintainers group needs and opinions.  Eventually, Unity will remove XRI Controller related stuff definitively but I guess I went too ahead :) ... Did I answer your questions?

Reply:
@ms-RistoRK Yes, thank you for answering.  It probably needs clarification from @AMollis , but I don't believe the "remove where possible" refers to any eventual removal or deletion of deprecated components (like XRI controllers) or even removal of them from the deprecated pre-XRI3 rig and prefabs for the upcoming release.  My understanding is that it only refers to removal of them from the **_new_** XRI3-based rig and prefabs - as in the new XRI3-based rigs and prefabs should seek to not use them where possible over time.

The legacy, pre-XRI3 rig and prefabs will continue containing these deprecated components and our next release will still contain them.  So no preparation for their eventual removal seems necessary at this time (besides the new XRI3-based rig setup and components) as our next release will continue containing them, likely for some time.